english/lightsaber-falls/lightsaber-falls.md

```diff
@@ -0,0 +1,9 @@
+You slice through the marshmallow man with your lightsaber. 
+Luckily for you, the heat from it cooks the marshmallows to perfection.
+You now have a tasty snack and a way out of the house. 
+
+But as you hurry to smoosh through the slowly closing hole in the wall, the lightsaber clatters to the floor.
+
+[You leap out nimbly and gaze back at the weapon](https://github.com/LaGracia/create-your-own-adventure/blob/lightsaber-falls/english/lightsaber-falls/wampa.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.